tests/commons/big_query/copy_job_async/copy_job/test_copy_job_service.py

```diff
@@ -258,7 +258,7 @@ def test_that_copy_table_should_create_correct_post_copy_action_if_access_denied
                     'state': 'DONE',
                     'errors': [
                         {
-                            'reason': 'Access Denied',
+                            'reason': 'Invalid',
```

Comment:
Why this message doesn't provide any information I believe? Could you provide me some explanations please, because commit name is confusing right now